README.md

```diff
@@ -93,6 +93,10 @@ The support for Apex analysis is only available together with SonarQube Enterpri
 
 The support for PL/SQL analysis is only available together with SonarQube Developer Edition or SonarCloud (see Connected Mode below). You also need the [Oracle Developer Tools for VSCode](https://marketplace.visualstudio.com/items?itemName=Oracle.oracledevtools) extension.
 
+### Injection Vulnerabilities specific requirements
+
+Injection vulnerabilities are not detected by local SonarLint analysis. In order to browse Injection Vulnerabilities in SonarLint for VSCode, [Connected Mode](https://github.com/SonarSource/sonarlint-vscode#connected-mode) with SonarQube Developer Edition or SonarCloud needs to be used. Once a Project Binding is configured, SonarLint will pull Injection Vulnerability issues detected for the remote project on SonarQube/SonarCloud.
```

Comment:
Maybe it is too much to strike the first sentence; if we keep it, I would recommend reframing it a little. Here is my recommendation:

To browse Injection Vulnerabilities in SonarLint for VSCode, [Connected Mode](https://github.com/SonarSource/sonarlint-vscode#connected-mode) with SonarQube Developer Edition or SonarCloud must be used. Once a Project Binding is configured, SonarLint will pull Injection Vulnerability issues detected from the remote project on SonarQube/SonarCloud.

Reply:
actually, does it also pull from the EE and DC SonarQube versions?
If yes, I would recommend option 2 below, but 3 is also ok. 

1. _SonarQube Developer Edition or SonarCloud_ 
2. _SonarQube Developer Edition and above, or SonarCloud_ 
3. _SonarCloud or SonarQube Developer Edition and above_